Problem1.py

```diff
@@ -0,0 +1,33 @@
+"""
+Time Complexity - O(n)
+Space Complexity - O(1)
+Using two pointers will traverse entire list and perform in place modification
+j will indicate the elements of result list
+initially count = 1
+traversing through list
+    if nums[i] == nums[i-1]
+        increment count
+    else
+        count = 1
+    if count <= 2
+        nums[j] = nums[i]
+        j += 1
+        """
+class Solution:
+    def removeDuplicates(self, nums: List[int]) -> int:
```

Comment:
This does not work on Leetcode. Please try to fix this. You can contact me, the TA or the professor for any questions/clarifications.